server/base/src/main/javadoc/META-INF/NOTICE

```diff
@@ -1,5 +0,0 @@
-Apache Accumulo
-Copyright 2011-2016 The Apache Software Foundation
```

Comment:
Ok, strange why we had added this in the first place (I assumed it was done because they weren't being placed automagically).